README.rst

```diff
@@ -12,12 +12,23 @@ Installation
     pip install polpo@git+https://github.com/geometric-intelligence/polpo.git@main
 
 
-Or, to also install all the dependencies:
```

Comment:
It is a "Or" here, as second does first + all dependencies installation.

Maybe we should add that `all` install too many dependencies and maybe a user should pick the optional dependencies (link to pyproject) that fit them?